client/lib/domains/index.js

```diff
@@ -13,7 +16,9 @@ import wpcom from 'lib/wp';
 import { type as domainTypes } from './constants';
 
 const GOOGLE_APPS_INVALID_TLDS = [ 'in' ],
-	GOOGLE_APPS_BANNED_PHRASES = [ 'google' ];
+	GOOGLE_APPS_BANNED_PHRASES = [ 'google' ],
+	OPENHRS_TLDS = [ '.wales' ],
+	OPENSRS_TLDS = [ '.live' ];
```

Comment:
I have missed this before, but this is not going to work a couple of weeks from now. Some of the TLDs will have different providers depending on certain conditions. We should export that information from domains API and consume it here.